src/drivers/dw/dma.c

```diff
@@ -67,7 +67,7 @@ struct dw_dma_chan_data {
 static const uint32_t burst_elems[] = {1, 2, 4, 8};
 
 #if CONFIG_HW_LLI
```

Comment:
@tlauda would really be nice to see a full explanation of why 3 linked items are not enough. This is a sensible area and good explanations will help people in the future to easily to debug.

Reply:
I'm guessing this is to give more overhead for FW to re-write HW LLIs if needed as the stream is running for state changes.